include/smack/SmackRep.h

```diff
@@ -150,6 +166,14 @@ class SmackRep {
   virtual const Stmt* alloca(llvm::AllocaInst& i);
   virtual const Stmt* memcpy(const llvm::MemCpyInst& msi);
   virtual const Stmt* memset(const llvm::MemSetInst& msi);
+  virtual const Stmt* loadAsBytes(const llvm::LoadInst& li);
+  virtual const Stmt* storeAsBytes(const llvm::StoreInst& si);
```

Comment:
To be more specific, load/store will return
Stmt::assign(mem(addr),expr(val))), i.e., $M.X(.iX)[addr] = val; for normal (non-bit-vector) and typed (i16, i32, etc.) memory access
Stmt::assign(expr(val), Expr::fn(opName("load", {getPtrSize(addr)}), Expr::id(memPath(r, 8)), expr(addr))), i.e., val = $load.iX($M.X(.iX), addr); for unsafe/byte-addressed memory address

Am I correct on it?